src/common/imageio_module.c

```diff
@@ -69,6 +69,8 @@ static int dt_imageio_load_module_format(dt_imageio_module_format_t *module, con
 {
   module->widget = NULL;
   module->parameter_lua_type = LUAA_INVALID_TYPE;
+  module->ready = FALSE;
```

Comment:
Please set to TRUE by default as this corresponds to current behavior. It will only be set to FALSE in avif.c if needed and all other setting can be removed. See below.

Reply:
Done.